protocols/pop3/src/main/java/org/apache/james/protocols/pop3/core/AbstractApopCmdHandler.java

```diff
@@ -81,7 +81,7 @@ public Response onCommand(POP3Session session, Request request) {
     
     @Override
     protected final Mailbox auth(POP3Session session, Username username, String password) throws Exception {
-        return auth(session, (String)session.getAttachment(POP3Session.APOP_TIMESTAMP, State.Connection), username, password);
+        return auth(session, session.getAttachment(POP3Session.APOP_TIMESTAMP, State.Connection).orElse(""), username, password);

```

Comment:
Why `""` before this refactoring the value was `null` ?
Can you define the default value as a constant too ?

Reply:
it's not used